README.md

```diff
@@ -48,151 +97,25 @@ For UI development purposes, we recommend using [npm][] directly. Please ensure
 	$ npm install
 	$ ng build --prod
 
-**NOTE:** Before building be sure that the `ng-serve` development server has been shutdown.
-
-### Build fails after merging a branch or changing branches
-
-In some cases the npm-modules or other dependencies may become inconsistent during branch changes.
-In order to resolve the problem we need to clean out inconsistent dependencies.
-The following instructions can be used to do this:
-
-* Shutdown the development server if it is running.
-* Commit or stash your changes
-* Execute the following:
-	* `git clean -fx`
-	* `npm install`
-* If you stashed your files execute the following:  `git stash pop`
-* Now build the application by executing the following: `ng build --prod`
-
-## Important Build-related Configuration Files
-
-* **pom.xml** Maven config file
-* ui/**package.json** Node dependencies
-
-# Running Tests - Locally
-
-## Unit Tests
-
-```bash
-	$ ng test --single-run
-```
-
-## E2E Tests
-
-```bash
-	$ npm run e2e
-```
-
-# Running Tests - SauceLabs
-
-Before you can run tests using SauceLabs, please setup your username and password:
-
-```bash
-    $ export SAUCE_USERNAME=your-username
-    $ export SAUCE_ACCESS_KEY=your-access-key
-```
-
-## Unit Tests
-
-```bash
-	$ npm run test-saucelabs-local
-```
-
-## E2E Tests
-
-For E2E tests, developers should refrain from using `localhost`. Instead, add `dataflow.local` to your local DNS hosts file.
-
-```bash
-	$ npm run e2e-saucelabs-local
-```
-
-## Slow Internet Connection
+Before building be sure that the `ng-serve` development server has been shutdown.<br >
+If you want to develop on the project, please, have a look on the [Developer Guideline](README_DEV.md/#development).
```

Comment:
Make the link to the developer guide more visible. Just bolden it should be sufficient I think.